cfme/infrastructure/virtual_machines.py

```diff
@@ -1473,7 +1473,15 @@ def step(self, *args, **kwargs):
 
 @navigator.register(Vm, 'candu')
 class VmUtilization(CFMENavigateStep):
-    VIEW = VMUtilizationAllView
+    @property
+    def VIEW(self):     # noqa
+        """Property returning VM Utilization Views as per provider type"""
+        if self.obj.provider.type is 'virtualcenter':
```

Comment:
**REQUIRED**: I'm also not keen on this - what we should do instead is have a property on the provider class that links to the view instead. If the views are inside the provider modules, this actually becomes pretty easy.